bin/tessel-2.js

```diff
@@ -115,7 +115,8 @@ parser.command('crash-reporter')
   .option('test', {
     flag: true,
     help: 'Test the Crash Reporter.'
-  });
+  })
+  .help('Crash Reporter enable/disable');
```

Comment:
How do you know if it's enabled or disabled? Do you run it with --enable or anything like that?